docs/src/main/sphinx/sql/set-time-zone.rst

```diff
@@ -0,0 +1,61 @@
+=============
+SET TIME ZONE
+=============
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    SET TIME ZONE LOCAL
+    SET TIME ZONE expression
+
+Description
+-----------
+
+Sets the current default time zone displacement for the current SQL-session.
+
+If the ```LOCAL`` option is specified, the time zone for the current SQL-session
+will be set to the original time zone of the current SQL-session.
+
+If the ``expression`` option is specified:
+
+- if the type of the ``expression`` is a string, the time zone for the current SQL-session is set to the corresponding region-based time zone ID or the corresponding zone offset.
```

Comment:
use * for bullet lists instead of -, wrap the sentence at 80 with indent on next lines